docs/src/examples/tables.md

```diff
@@ -0,0 +1,98 @@
+# Tables
+
+`Table`s are coordinates in a tabular format (essentially a matrix), optionally with named columns. They have various constructors, for direct construction and also for conversion from other types.
+
+```jl
+import PGFPlotsX
+const pgf = PGFPlotsX
+using LaTeXStrings
+```
+
+```@setup pgf
+import PGFPlotsX
+const pgf = PGFPlotsX
+using LaTeXStrings
+savefigs = (figname, obj) -> begin
+    pgf.save(figname * ".pdf", obj)
+    run(`pdf2svg $(figname * ".pdf") $(figname * ".svg")`)
+    pgf.save(figname * ".tex", obj);
+    return nothing
+end
+```
+
+Let
+```jl
+x = linspace(0, 2*pi, 100)
+y = sin.(x)
+```
+
+```@setup pgf
+x = linspace(0, 2*pi, 100)
+y = sin.(x)
+```
+
+You can pass these coordinates in unnamed columns:
+
+```@example pgf
+pgf.Plot(pgf.Table([x, y]); incremental = false)
+savefigs("table-unnamed-columns", ans) # hide
+```
+
+[\[.pdf\]](table-unnamed-columns.pdf), [\[generated .tex\]](table-unnamed-columns.tex)
+
+![](table-unnamed-columns.svg)
+
+or named columns:
+
+```@example pgf
+pgf.Plot(pgf.Table([:x => x, :y => y]); incremental = false)
+savefigs("table-named-columns", ans) # hide
+```
+
+[\[.pdf\]](table-named-columns.pdf), [\[generated .tex\]](table-named-columns.tex)
+
+![](table-named-columns.svg)
+
+or a dictionary, here renamed using options:
```

Comment:
What's the reason for including a `Dict` constructor. Since the ordering is unknown, is that a good idea?